db/seeds.rb

```diff
@@ -18,8 +18,14 @@
 StoreMembership.create(store: demo_store, user: admin_user, role: :admin)
 StoreMembership.create(store: demo_store, user: regular_user, role: :regular)
 
-puts "Creating manufacturers, providers"
+puts "Creating manufacturers, providers, categories"
 5.times do
   Manufacturer.create(name: Faker::HarryPotter.character, store: demo_store)
   Provider.create(name: Faker::RickAndMorty.character, store: demo_store)
 end
+
+category = Category.create(label: Faker::Space.planet, store: demo_store, tva: :standard_rate)
+Category.create(label: Faker::Space.planet, store: demo_store, tva: :reduced_rate, category: category)
+Category.create(label: Faker::Space.planet, store: demo_store, tva: :reduced_rate_alt)
+Category.create(label: Faker::Space.planet, store: demo_store, tva: :super_reduced_rate)
+Category.create(label: Faker::Space.planet, store: demo_store, tva: :parking_rate, category: category)
```

Comment:
User .sample for random TVA and category. You can pass nil as value so no category is applied